Sources/Tools/MaxMain/MaxCompat.h

```diff
@@ -291,6 +291,12 @@ class plMaxObject : public plMaxAnimatable<T>
 
 // Old versions of Max define this as an integer, not a Class_ID
 #define XREFOBJ_COMPAT_CLASS_ID Class_ID(0x92aab38c, 0)
+// This definition is removed in later versions of the sdk
+// In newer versions of the sdk Progress returns an enum value of KeyReduceResult
+#if MAX_VERSION_MAJOR <= 25
+#   define BIPDRIVEN_CONTROL_CLASS_ID BIPSLAVE_CONTROL_CLASS_ID
+#   define KeyReduceResult int
```

Comment:
What was the reason for switching from `using` to `#define` here?

```suggestion
    typedef KeyReduceResult int;
```

Reply:
It broke the build in the initial attempt with `using`, but that might have been due to the MAX_VERSION_MAJOR check being wrong (apparently they skipped a number somewhere between years)

Reply:
yea the define was an attempt to "fix" the issue